### PR TITLE
Remove the breadcrumb if jetpack in campaign list (blazepress)

### PR DESCRIPTION
**Pull request by jjolmo**
Related to #

## Proposed Changes

* Remove the breadcrumb and back button in BlazePress campaign details if the app is Jetpack

## Testing Instructions
* Open a campaign in calypso (no jetpack). It should display the breadcrumb. If you put the browser in mobile it will display the "back" button
* Open a campaign in Jetpack. It should not display the breadcrumb and the back button

## Pre-merge Checklist

- [x] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [x] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [x] Have you checked for TypeScript, React or other console errors?
- [x] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx [hunk 1]
```diff
@@ -254,13 +254,17 @@ export default function CampaignItemDetails( props: Props ) {
...
 			<header className="campaign-item-header">
 				<div>
-					<div className="campaign-item-breadcrumb">
-						{ ! isLoading ? (
-							<Breadcrumb items={ navigationItems as Item[] } compact={ isSmallScreen } />
-						) : (
-							<FlexibleSkeleton />
-						) }
-					</div>
+					{ ! isRunningInJetpack ? (
```
Comment (sbarbosa) [inline client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx]: The `isRunningInJetpack` flag is used in the Jetpack Dashboard app, not the Jetpack mobile app. The mobile app loads a webview of the WordPress.com site. 

To hide something for the Jetpack App you need to use the function `isWpMobileApp`.

Also, you can hide elements using CSS too (if no additional code is needed to run). You have this global class to do it: `.is-mobile-app-view`
Check the file `client/my-sites/promote-post-i2/style.scss`

**Review by sbarbosa — CHANGES_REQUESTED** [review]
Hi @jjolmo! I think you meant to check for the mobile app here, instead for the Jetpack Dashboard. I added a comment on how to check that. 

To test it you need to create in Chrome a custom device using a specific user agent.
I use this one: `Mozilla/5.0 (iPhone; CPU iPhone OS 12_1 like Mac OS X) AppleWebKit/605.1.15 (KHTML, like Gecko) Mobile/16B91 wp-iphone/12.1`

File: client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx (hunk 1 above)
Comment (jjolmo) [reply client/my-sites/promote-post-i2/components/campaign-item-details/index.tsx]: Gotcha. Many thanks. I'll do the change

**Comment by jjolmo** [comment]
@sbarbosa I've fixed it and I've also removed the header size in  wp mobile app since we don't have the header there

**Review by sbarbosa — APPROVED** [review]
Thanks for the change @jjolmo!
I tested it, and it's working fine. I find a minor design bug in the Jetpack mobile version of the details page. We have a min-height in place for the header, and without the breadcrumb, there is an extra white space at the bottom.

**Mobile (Jetpack App):**
Screenshot 2023-07-31 at 09 33 46

**Mobile (Web):**
Screenshot 2023-07-31 at 09 34 37

**Comment by jjolmo** [comment]
good catch @sbarbosa . Changed!

**Pull request merged**